Harmony/Public/Patching/HarmonyManipulator.cs

```diff
@@ -528,6 +533,28 @@ bool WriteFinalizerCalls(bool suppressExceptions)
 				if (modifiesControlFlow)
 					lastInstruction.OpCode = OpCodes.Ret;
 
+				foreach (var ilman in ilmanipulators)
```

Comment:
This whole code should be in its own separate method like with other patch types